EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -552,7 +626,7 @@ private static float GetMobilityZScore(PeptideSpectralMatch psm, PeptideWithSetM
                                 label = false;
                                 newPsmData = CreateOnePsmDataEntry(searchType, fileSpecificParameters, psm, sequenceToPsmCount, timeDependantHydrophobicityAverageAndDeviation_unmodified, timeDependantHydrophobicityAverageAndDeviation_modified, fileSpecificMedianFragmentMassErrors, chargeStateMode, csm.BestMatchingPeptides.First().Peptide, 0, label);
                             }
-                            else if (!csm.IsDecoy && !csm.BetaPeptide.IsDecoy && psm.FdrInfo.QValue <= 0.01)
+                            else if (!csm.IsDecoy && !csm.BetaPeptide.IsDecoy && psm.FdrInfo.QValue <= 0.0005)
```

Comment:
Why change QValue to 0.0005?

Reply:
instead of training with 25%, i'm now training with 75%. So, i thought i could change the cutoff and train on better values. moving from 0.01 to 0.005 only eliminates a small fraction of PSMs, but it eliminates the worst of the PSMs below 1%.